jupyter_telemetry/eventlog.py

```diff
@@ -124,7 +124,7 @@ def register_schema(self, schema):
 
         self.schemas[(schema['$id'], schema['version'])] = schema
 
-    def record_event(self, schema_name, version, event):
```

Comment:
I think this method's signature is getting big enough now that we might want to add documentation for each argument to the docstring... It doesn't have to be in this PR. I was thinking about doing a documentation audit sometime soon.

Reply:
+1 for documentation audit soon!